Tiltfile

```diff
@@ -739,6 +758,32 @@ def allow_tcp_udp_ports():
         allow_parallel = True,
     )
 
+def needs_vnet_peering(flavor_name):
+    """
+    Check if the flavor requires VNet peering configuration.
+
+    Args:
+        flavor_name (str): The name of the flavor to check
+
+    Returns:
+        bool: True if the flavor needs VNet peering, False otherwise
+
+    Flavors requiring VNet peering are:
+    - apiserver-ilb
+    - windows-apiserver-ilb
+    - aks
+    """
+    flavors_needing_vnet_peering = [
+        "apiserver-ilb",
+        "windows-apiserver-ilb",
+    ]
```

Comment:
We probably want to get this exported or updated so that 1P users can add their own flavors that need vnet peerings. 
I will open up a good-first-issue to add this.